src/web/components/Onwards/OnwardsUpper.tsx

```diff
@@ -1,7 +1,18 @@
-import React from 'react';
+import React, { Suspense } from 'react';
 
 import { joinUrl } from '@root/src/web/lib/joinUrl';
-import { OnwardsData } from './OnwardsData';
+import { initPerf } from '@root/src/web/browser/initPerf';
+
+const OnwardsData = React.lazy(() => {
+    const { start, end } = initPerf('OnwardsData');
+    start();
+    return import(/* webpackChunkName: "OnwardsData" */ './OnwardsData').then(
+        module => {
+            end();
+            return { default: module.OnwardsData };
+        },
+    );
+});
```

Comment:
Did you mean to keep this in here?

I think it's right that we lazy load the OnwardsUpper component only when the reader scrolls the content into view. It's a lot less clear if this secondary split adds benefit.

We know without doubt we only want to load based on scroll position but beyond that it could be argued that most articles have onwards content so splitting here might possibly slow things down for the majority of readers? Tbh, I'm not sure. It'd be great to keep this change separate where we can discuss and test it in isolation?

Reply:
^^ I was removing it